src/viam/api/config/viam-cpp-sdk-libviamapi.pc.in

```diff
@@ -7,7 +7,7 @@ Name: @PROJECT_NAME@-libapi
 Description: @PROJECT_DESCRIPTION@
 URL: @PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
-Requires: gRPC++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@
+Requires: grpc++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@
```

Comment:
I was surprsied to find this was making the `pkg-config` call fail on debian, see:
https://github.com/viamrobotics/viam-cpp-sdk/actions/runs/10272978914/job/28426411691#step:4:2749

comparing both my mac and the Debian.bullseye docker image, it seems that the file is indeed called:

```
/opt/homebrew/Cellar/grpc/1.62.2/lib/pkgconfig/grpc++.pc
/usr/lib/aarch64-linux-gnu/pkgconfig/grpc++.pc
```
Perhaps even more suprising, it seems like this worked on my mac because `pkg-config` there was case insensitive. In any case I think this is the more correct spelling of it going forward

Reply:
Mac file system is case-insensitive :/